composer.json

```diff
@@ -15,7 +15,7 @@
         "ocramius/proxy-manager": "~1.0 || ~2.0",
         "symfony/proxy-manager-bridge": "^2.8.40 || ^3.4.11",
         "zetacomponents/system-information": "^1.1.1",
-        "ezsystems/ezplatform-admin-ui": "^1.5@dev"
+        "ezsystems/ezplatform-admin-ui": "dev-ibx-385-responsiveness-if-composer-files-corrupted as 1.5.x-dev"
```

Comment:
Friendly reminder: revert this change before merge

Reply:
It's mentioned in the PR description.